approaches/bulk-data-loading/src/Constants.js

```diff
@@ -0,0 +1,28 @@
+export const filename = "../data/large-data-set.csv";
+
+export const inventoryRecordSize = 25000
+export const batchSize = 100;
+export const clientConcurrency = 25
```

Comment:
When running locally can you measure how many requests/s this generates?

Reply:
I maxed out around 32 requests/s.  `clientConcurrency` of 50, 100, and 200 yielded just about the same throughput results.